projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts

```diff
@@ -1,26 +1,101 @@
 import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
-import { EventSpy } from '@stencil/core/dist/declarations';
 
 describe('dot-label', () => {
     let page: E2EPage;
     let element: E2EElement;
-    let input: E2EElement;
-
-    beforeEach(async () => {
-        page = await newE2EPage({
-            html: `
-            <dot-label
-                label='Address:'
-                name='Address'
-                value='Address'>
-            </dot-textarea>`
+
+    const getLabel = async () => await page.find('label');
+    const getText = async () => await page.find('.dot-label__text');
+    const getMark = async () => await page.find('.dot-label__required-mark');
+
+    describe('<slot />', () => {
+        beforeEach(async () => {
+            page = await newE2EPage({
+                html: `
+                <dot-label labe="hello world">
+                    <h1>into the slot</h1>
+                </dot-label>`
+            });
+            element = await page.find('dot-label');
```

Comment:
use the function getLabel

Reply:
Is not the same, the `getLabel` return the `<label>` inside `<dot-label>`